src/components/Layout/Layout.jsx

```diff
@@ -0,0 +1,14 @@
+import Footer from "../Footer/Footer";
+import Header from "../Header/Header";
+
+const Layout = ({ children }) => {
+  return (
+    <>
+      <Header />
```

Comment:
Please wrap wrap layout header, children and footer and add dummy state to layout component

Reply:
Ok to go.